### PR TITLE
Update showbiz links

### PR DESCRIPTION
**Pull request by surma**
Split this in 2 commits for easy review:

- 1st commit just updates the links to demos and code since we moved the repository from GoogleChrome to GoogleChromeLabs
- 2nd commit re-formats so `gulp test` complains as little as possible

### CODE REVIEW

File: src/content/en/updates/2016/12/performant-parallaxing.md [hunk 1]
```diff
@@ -137,34 +232,74 @@ By applying `position: -webkit-sticky` to the parallaxing element we can effecti
...
-* **With** `position: sticky`, the nearer the element is to z=0 the **less** it moves.
-* **Without** `position: sticky`, the nearer the element is to z=0 the **more** it moves.
+* **With** `position: sticky`, the nearer the element is to z=0 the **less** it
+  moves.
+* **Without** `position: sticky`, the nearer the element is to z=0 the **more**
+  it moves.
 
-If that all seems a bit abstract, [take a look at this demo](http://jsbin.com/sexiwe/edit?html,css,js,output) by Robert Flack, which demonstrates how elements behave differently with and without sticky positioning. To see the difference you need Chrome Canary (which is version 56 at the time of writing) or Safari.
+If that all seems a bit abstract, [take a look at this
+demo](http://jsbin.com/sexiwe/edit?html,css,js,output) by Robert Flack, which
```
Comment (petele) [inline src/content/en/updates/2016/12/performant-parallaxing.md]: nit - https?

File: src/content/en/updates/2016/12/performant-parallaxing.md [hunk 2]
```diff
@@ -137,34 +232,74 @@ By applying `position: -webkit-sticky` to the parallaxing element we can effecti
...
+demonstrates how elements behave differently with and without sticky
+positioning. To see the difference you need Chrome Canary (which is version 56
+at the time of writing) or Safari.
 
 <a href="http://jsbin.com/sexiwe/edit?output">
```
Comment (petele) [inline src/content/en/updates/2016/12/performant-parallaxing.md]: nit - https

**Comment by surma** [comment]
@petele fix’d. PTAL

**Comment by petele** [comment]
@surma confirmed all changes were formatting, breaking lines.

**Pull request merged**